core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/signup.jelly

```diff
@@ -188,15 +188,15 @@ THE SOFTWARE.
 
                                     const passwordStrength = function(score) {
                                         if (score > 80) {
-                                            return "Strong";
+                                            return "${%Strong}";
```

Comment:
Didn't know it works in Javascript. Nice to know

Reply:
It's ok, I already test.